Hexapod/v1/default.yaml

```diff
@@ -0,0 +1,42 @@
+camera_config:
+  elevation_coeffs:
+    # Fit of https://github.com/bxin/hexrot/blob/master/LUT/Camera%20Hexapod%20Motions%20in%20Elevation%20Axis%202020%2007%2023.xlsx
+    # commit 0513b27 2020-08-04
+    - [0]
+    - [6.41754532e+02, 5.52880420e-01, -2.06081416e-01, 1.14135938e-04, 3.68377168e-06]
+    - [-6.05001038e+02, 1.58266699e+01, 4.76907398e-03, -1.00806469e-03, 2.63260121e-06]
+    - [1.77291858e-02, -2.16979613e-07, -5.39973300e-06, -1.50534189e-10, 1.41927682e-10, -7.00491997e-14, -9.63132987e-16]
```

Comment:
I think it would be better if we use 5th order standard polynomials everywhere (at least for now). It looks messy if we use different number of coefficients for each axis. It is OK to have a little bit of overkill sometimes - we are only dealing with 6 numbers for each 5th order polynomial, which is not a lot. Analysis shows that 5th order polynomials would be accurate enough for all cases. Note that what we need to use to evaluate fitting errors are the corresponding non-repeating errors along each axis.

Reply:
I will increase the order for y and z, but I am against trimming u since the extra coefficient does help the fit. I don't think it looks neater is sufficient justification to trim.

Reply:
As we talked about on Slack - the justifications are (1) consistency between different AOS components, (2) LTS-206 sec 3.7.2.3.5 (3) 5th order standard polynomial is simple and shown to be accurate enough.